sdk/purview/azure-purview-catalog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0b2 (2021-09-24)
```

Comment:
Can you mention the breaking changes in the changelog? The ones I can see are
1. No more rest layer, no more request builders, we have convenience operations for you. You can even have a little migration guide, say instead of this code
```python
...
from azure.purview.catalog.rest import types
request = types.build_get_all_type_definitions_request()
response = client.send_request(request)
```
you have this code
```python
response = client.types.get_all_type_definitions()
```
2. The HttpRequest parameter to `send_request` has changed from `http_request` to `request`
3. Ordering of endpoint and credential params have changed

New features are 
1. We have convenience operations now on the client
2. The service api version we're targeting has gone from 2021-05 -> 09